src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -2,17 +2,23 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.CompilerServices;
+
 internal static partial class Interop
 {
     internal static partial class Libraries
     {
         // Shims
         internal const string SystemNative = "System.Native";
-        internal const string GlobalizationNative = "System.Globalization.Native";
         internal const string NetSecurityNative = "System.Net.Security.Native";
         internal const string CryptoNative = "System.Security.Cryptography.Native.OpenSsl";
         internal const string CompressionNative = "System.IO.Compression.Native";
         internal const string IOPortsNative = "System.IO.Ports.Native";
         internal const string Libdl = "libdl";
+#if CORECLR
```

Comment:
There is no single `#if CORECLR` under libraries/common. I think `#if CORECLR` under libraries/common may get confusing very fast. I am wondering whether we should rather recognize `System.Globalization.Native` as a special library in the runtime, and leave this one intact.

It is a single place in dllimport.cpp:

```
    if (pNMD->GetLoaderModule()->IsSystem() && (strcmp(szLibName, "QCall") == 0 || strcmp(szLibName, "System.Globalization.Native"))
    {
        ndirectflags |= NDirectMethodDesc::kIsQCall;
    }
```

Reply:
Sounds reasonable, but wouldn't that make it harder for people when reading the Interop code to realize it is a QCall? Or it doesn't really matter?

Reply:
Also, does that mean that when we link this into mono, we would have to special case it in mono runtime as well?

Reply:
> Or it doesn't really matter?

I do not think it matters. When you are reading the code, you typically do not care how it is stitched together - you only care about that when reading, e.g. cmake file that links everything together.

> Also, does that mean that when we link this into mono, we would have to special case it in mono runtime as well?

Yes, I expect that it would be special-cased somehow. I do not know what the most natural solution for Mono would be.

Reply:
Ok, sounds good. Then I'll special case this in the runtime directly.